armi/settings/tests/old_xml_settings_input.xml

```diff
@@ -2,7 +2,6 @@
 
 <settings version="v6.1.5">
 	<aclpMaterial value="SS316" />
-	<armiLocation value="\\albert\apps\dev\armi\master" />
 	<axialExpansion value="True" />
 	<bondRemoval value="True" />
 	<branchVerbosity value="extra" />
```

Comment:
Any reason we can't just delete this? Are any tests still using it? I'd assume not, since it contains settings not defined by ARMI....

Reply:
There is a unit test that still uses this. It is curious that the conversion still worked with these settings there.